shared/utils/urls.test.ts

```diff
@@ -0,0 +1,81 @@
+import * as urlsUtils from "./urls";
+
+describe("IsUrl Method", () => {
+  describe("invalid urls", () => {
+    it("should return false", () => {
+      expect(urlsUtils.isUrl("")).toEqual(false);
```

Comment:
[`.toBe(value)`](https://jestjs.io/docs/expect#tobevalue) should just be fine for primitive values. Handy as well.